tests/meta-mender-beaglebone-yocto-ci/recipes-mender/mender-qa/mender-qa.bbappend

```diff
@@ -1,13 +1,15 @@
-PACKAGES += "rssh-tunnel"
-FILESEXTRAPATHS_prepend := "${THISDIR}/files/beagleboneblack/:${THISDIR}/files/:"
+PACKAGES_append_mender-client-enabled = " rssh-tunnel"
+FILESEXTRAPATHS_prepend_mender-client-enabled := "${THISDIR}/files/beagleboneblack/:${THISDIR}/files/:"
```

Comment:
No one else will use these. Is it really necessary to add all these overrides for test layers? Do we have to qualify as Yocto compatible on a repository level, or on a layer level?

Reply:
I debated that and it's probably ok not to handle these layers.  I was even thinking of adding a mender-testing-enabled feature.  In my case, I try to keep all the layers in my normal config and simply enable/disable things in local.conf.  I try to avoid having multiple build directories and even started using a single build directory for multiple MACHINEs.  I can see some value in having the testing layers enabled in our debug builds but simply disabled in local.conf unless needed.

Your call though as I don't get into these layers much currently.

Reply:
That's a good point, I guess that's a good way to experience first hand whether the layer is making intrusive changes or not! I'm ok with leaving it as is, except possibly changing the name of the override, as discussed elsewhere.

Reply:
Should I create a mender-testing-enabled override specifically for these settings?

Reply:
Looks like I tried to do that but just missed it in a couple of places.

Reply:
I think yes. But this requires a change in the build scripts too, to [add the feature when the testing layers are being used](https://github.com/mendersoftware/mender-qa/blob/2399567768f42186ab3cdfb28f442fe82ab06e2c/scripts/jenkins-yoctobuild-build.sh#L554).